rasa/core/nlg/template.py

```diff
@@ -103,12 +103,26 @@ def _fill_template(
         # Getting the slot values in the template variables
         template_vars = self._template_variables(filled_slots, kwargs)
 
-        # Filling the template variables in the template text
-        if template_vars:
-            if "text" in template:
-                template["text"] = interpolate(template["text"], template_vars)
-            elif "custom" in template:
-                template["custom"] = interpolate(template["custom"], template_vars)
+        for key in ["text", "image", "custom"]:
+            # Filling the template variables in the template
+            if template_vars and (key in template):
+                try:
```

Comment:
wow, there were quite some changes in the meanwhile. I think we don't have to the comment and the exception handling in there anymore.

How about:

```python
# Filling the template variables in the template text
        if template_vars:
            keys_to_interpolate = ["text", "image", "custom"]
            for key in keys_to_interpolate:

                if key in template:
                    template[key] = interpolate(template[key], template_vars)
```

Reply:
> 
> 
> wow, there were quite some changes in the meanwhile. I think we don't have to the comment and the exception handling in there anymore.
> 
> How about:
> 
> ```python
> # Filling the template variables in the template text
>         if template_vars:
>             keys_to_interpolate = ["text", "image", "custom"]
>             for key in keys_to_interpolate:
> 
>                 if key in template:
>                     template[key] = interpolate(template[key], template_vars)
> ```

thanks @wochinge.